source/components/ChooseHouseSection.js

```diff
@@ -0,0 +1,93 @@
+import { normalize } from '../js/utils.js';
+
+class ChooseHouseSection extends HTMLElement {
+  constructor() {
+    super();
+    this.attachShadow({ mode: 'open' });
+
+    // TODO: change px to rem
+    const style = new CSSStyleSheet();
+    style.replaceSync(`
+        :host {
+            height: 100%;
+            display: grid;
+            grid-template-columns: repeat(1, minmax(0, 1fr));
+            gap: 0.25rem;
+        }
+
+        
+        .wrapper {
+            text-align: center;
+        }
+
+        .wrapper h1 {
+            font-family: 'Almendra SC';
+            font-style: normal;
+            font-weight: 400;
+            font-size: 5rem;
+            line-height: 6rem;
+            text-align: center;
+
+            color: #FFFFFF;
+        }
+
+
+        .row{
+            display: flex;
+            justify-content: center;
+            align-items: center;
+        }
+
+        .option {
+            gap: 3rem;
+            border-radius: 1rem;
+            padding: 4rem 5rem;
```

Comment:
these are really large numbers, maybe we can make them smaller if we want to be mobile compatible